cache_test.go

```diff
@@ -119,3 +127,186 @@ func TestCacheWithSubCache(t *testing.T) {
 
 	dosCache(c, t, 1000)
 }
+
+func TestRedisCacheWithCompressionJSON(t *testing.T) {
+	redisClient, err := InitRedis()
+	if err != nil {
+		t.Skipf("skipping because of redis error: %s", err.Error())
+	}
+
+	engine, err := compression.NewEngine(1)
```

Comment:
could we use compression.ZstdCompressionService.GetID() instead of the '1' constant?